src/__tests__/index.js

```diff
@@ -468,17 +459,13 @@ test('can accept functions which return string class names', () => {
 test('should accept user defined contextTypes', () => {
   const dynamicStyles = jest.fn()
   const Child = glamorous.div(dynamicStyles)
-  Child.contextTypes = {
-    fontSize: PropTypes.number,
-  }
+  Child.contextTypes = {fontSize: PropTypes.number}
 
-  const context = {
-    fontSize: 24,
-  }
+  const context = {fontSize: 24}
 
   render(<Child />, {context})
   expect(dynamicStyles).toHaveBeenCalledTimes(1)
   const theme = {}
-  const props = {}
+  const props = {theme}
```

Comment:
This would also move `context` to second arg which matches stateless components. 👍